### ISSUE TITLE
Licensing unclear (GPL2 or GPL2+?)

### ISSUE THREAD

**Issue by lukegb** [AirenSoft/OvenMediaEngine#250]
Since the entirety of the GPL2 LICENSE file is included but there are no other license headers that I can see, it's not clear if this project is intended to be licensed under GPL2 *only*, or GPL2 "or any later version".

Per the LICENSE file:

```
  If you develop a new program, and you want it to be of the greatest
possible use to the public, the best way to achieve this is to make it
free software which everyone can redistribute and change under these terms.

  To do so, attach the following notices to the program.  It is safest
to attach them to the start of each source file to most effectively
convey the exclusion of warranty; and each file should have at least
the "copyright" line and a pointer to where the full notice is found.

    <one line to give the program's name and a brief idea of what it does.>
    Copyright (C) <year>  <name of author>

    This program is free software; you can redistribute it and/or modify
    it under the terms of the GNU General Public License as published by
    the Free Software Foundation; either version 2 of the License, or
    (at your option) any later version.

    This program is distributed in the hope that it will be useful,
    but WITHOUT ANY WARRANTY; without even the implied warranty of
    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
    GNU General Public License for more details.

    You should have received a copy of the GNU General Public License along
    with this program; if not, write to the Free Software Foundation, Inc.,
    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
```

Would you mind clarifying which of these is intended? GPL2-only, or GPL2-or-any-later-version?

**Comment by dimiden** [comment]
@lukegb 
**TL;DR;** OME is licensed under the GPLv2 or later.

When we started the OME project, we decided to use GPLv2 and copied the license text from http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt (you are viewing this document).

To answer this question clearly, We thought about licensing again, and added ```or later``` explicitly to README because there is no reason not to support GPLv3.
